tests/spdx3/fixtures.py

```diff
@@ -298,6 +309,42 @@ def namespace_map_fixture(
     "relationship_type": RelationshipType.FIXED_IN,
 }
 
+ARTIFACT_DICT = {
+    "originated_by": ["https://spdx.test/tools-python/originatedBy"],
+    "supplied_by": ["https://spdx.test/tools-python/suppliedBy"],
+    "built_time": datetime(2004, 1, 1),
+    "release_time": datetime(2005, 1, 1),
+    "valid_until_time": datetime(2006, 1, 1),
+    "standard": ["https://spdx.test/tools-python/standard"],
+}
+
+SOFTWARE_ARTIFACT_DICT = {
+    "content_identifier": "contentIidentifier",
+    "purpose": SoftwarePurpose.OTHER,
+    "concluded_license": LicenseField,
+    "declared_license": LicenseField,
```

Comment:
This is not a correct value for these fields. But I suppose you ran into the circle definition of the `License` fixture being referenced in the same dictionary as the software artifact fixtures. I'll think about how to fix this, in the meantime we can merge it like this, I think.

Reply:
yes

Reply:
Also, do the values in SNIPPET_DICT assumed correctly, or need modifications?

Reply:
They are fine as far as I'm concerned! :)